codec-http/src/main/java/io/netty/handler/codec/http/cookie/DefaultCookie.java

```diff
@@ -156,15 +156,15 @@ public boolean equals(Object o) {
         } else if (that.domain() == null) {
             return false;
         } else {
-            return domain().equalsIgnoreCase(that.domain());
+            return domain().equals(that.domain());
```

Comment:
I know the spec doesn't say anything here, but it feels very weird to me that domain would be case sensitive. After all, it's more or less the host name of the server that originally set the cookie. And host names are definitively case insensitive. I would vote to keep case insensitivity here.